<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DNCLoggerMainScreen</class>
 <widget class="QMainWindow" name="DNCLoggerMainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nectar DNC Logger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>n-c-tools-120x120.png</normaloff>n-c-tools-120x120.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>791</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>363</y>
      <width>791</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>791</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>791</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5,5,5,5,5">
     <property name="spacing">
      <number>17</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="sendBtn">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseBtn">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resumeBtn">
       <property name="text">
        <string>Resume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortBtn">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartBtn">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backBtn">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="fileDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>351</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
